src/discussion/final-thoughts.tex

```diff
@@ -1,4 +1,4 @@
-\subsection{Final thoughts}\label{subsec:final-thoughts}
+\section{Final thoughts}\label{sec:final-thoughts}
```

Comment:
If we make subsections into sections we need to also change the folder structure. Each section should have its own folder with subsequent index.tex file

Reply:
Ahh sorry about that, still not used to the new structure. Will fix when I get back home

Reply:
@MadsHeilmann what should we do with the conclusion? Keep it as is or move it to openings? It's a chapter without sections